code/modules/hydroponics/prehistoric_plants.dm

```diff
@@ -9,7 +9,7 @@
 	packet_icon = "seed-telriis"
 	plant_icon = "telriis"
 	harvest_repeat = 1
-	chems = list(NUTRIMENT = list(1,10),PWINE = list(0,2))
+	chems = list(DIETHYLAMINE = list(0,10))
```

Comment:
And how else am I supposed go get my Poison Wine??

![image](https://user-images.githubusercontent.com/7573912/42443942-20f306ae-836f-11e8-8332-47d0cbd33c8e.png)

Reply:
It's a premium item at the bar vendor as of this PR

Reply:
I like PWINE farms...